mizar/common/rpc.py

```diff
@@ -21,13 +21,15 @@
 
 import logging
 import json
+import netifaces
+import os
 from mizar.common.common import run_cmd
 
 logger = logging.getLogger()
 
 
 class TrnRpc:
-    def __init__(self, ip, mac, itf='eth0', benchmark=False):
+    def __init__(self, ip, mac, itf=os.getenv("INET"), benchmark=False):
```

Comment:
Let's set the default value as "eth0" here: 
itf=os.getenv("INET") if os.getenv("INET") is not None else "eth0"